src/Application.vue

```diff
@@ -205,46 +207,49 @@ export default {
         this.isLoading = false
       })
 
-      // Automatically create a theme instance if one doesn't exist
-      if (themeWithoutInstances == themes.length) {
-        const flipletTheme = _.find(themes, { name: FLIPLET_THEME })
-
-        // Checks for older versions
-        ThemeModel.getAllVersions()
-          .then((result) => {
-            const allThemes = result.widgets
-            const versionOneTheme = _.find(allThemes, { name: 'Bootstrap', version: '1.0.0' })
-
-            if (!versionOneTheme.instances.length) {
-              return
-            }
-
-            // Save the old settings
-            this.oldThemeSettings = versionOneTheme.instances[0].settings
-            return versionOneTheme.instances[0].id
-          })
-          .then((id) => {
-            if (!id) {
-              return
-            }
-
-            return ThemeModel.delete(id)
-          })
-          .then(() => {
-            return this.createDefaultInstance(flipletTheme.id, toReuse)
-          })
-          .then(() => {
-            this.initialize(widgetData)
-          })
-          .then(this.reloadPagePreview)
-          .then(() => {
-            bus.$emit('saved-fields-set')
-          })
-          .catch((err) => {
-            this.error = Fliplet.parseError(err)
-            console.error(err)
-          })
-      }
+      Promise.all(themePromises)
+        .then(() => {
+          // Automatically create a theme instance if one doesn't exist
+          if (themeWithoutInstances == themes.length) {
```

Comment:
can't you just return from this block condition instead of having everything indented inside it?

```
if (false) {
  return
}

// do rest
```